include/SugarFields/Fields/CronSchedule/SugarFieldCronSchedule.php

```diff
@@ -1,6 +1,28 @@
 <?php
 require_once('include/SugarFields/Fields/Base/SugarFieldBase.php');
 
+function format_weekday(DateTime $date) {
```

Comment:
I see some design issue here about the global function. For e.g If these function are here and I want to reuse them, I still have to include the SugarFieldCronSchedule. These global function could be a part of the SugarFieldBase or could be an extension over the DateTime, also could be a part of the SugarFieldCronSchedule. Or worst but still better if these are in a separated php include like sugar_utils. (Also please try to avoid using global variables if it's possible - I'm thinking of 'current_language' in this case)